docs/DeveloperGuide.adoc

```diff
@@ -103,6 +104,7 @@ Having both Travis and AppVeyor ensures your App works on both Unix-based platfo
 When you are ready to start coding, you may move on to <<Design-Architecture>> to get some sense of the overall design.
 
 == Design
+This section depicts the implementation of the application, from UI to storage.
```

Comment:
- 'Describes', not 'depicts'. 
- The meaning of "from UI to storage" is also unclear at this point in the dev guide. Can remove.